airbyte-integrations/connectors/source-iterable/source_iterable/manifest.yaml

```diff
@@ -0,0 +1,504 @@
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - start_date
+      - api_key
+    properties:
+      api_key:
+        type: string
+        order: 1
+        title: API Key
+        airbyte_secret: true
+      start_date:
+        type: string
+        order: 0
+        title: Start date
+        format: date-time
+        examples: ["2021-04-01T00:00:00Z"]
+        pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$
+    additionalProperties: true
+type: DeclarativeSource
+check:
+  type: CheckStream
+  stream_names:
+    - lists
+streams:
+  - name: lists
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      paginator:
+        type: NoPagination
+      requester:
+        path: lists
+        type: HttpRequester
+        url_base: https://api.iterable.com/api/
+        http_method: GET
+        authenticator:
+          type: ApiKeyAuthenticator
+          api_token: "{{ config['api_key'] }}"
+          inject_into:
+            type: RequestOption
+            field_name: Api-Key
+            inject_into: header
+        error_handler:
+          type: CompositeErrorHandler
+          error_handlers:
+            - type: DefaultErrorHandler
+              response_filters:
+                - type: HttpResponseFilter
+                  action: IGNORE
+                  http_codes:
+                    - 401
+        request_headers: {}
+        request_body_json: {}
+        request_parameters: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - lists
+      partition_router: []
+    primary_key:
+      - id
+  - name: list_users
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      paginator:
+        type: NoPagination
+      requester:
+        path: lists/getUsers
+        type: HttpRequester
+        url_base: https://api.iterable.com/api/
+        http_method: GET
+        authenticator:
+          type: ApiKeyAuthenticator
+          api_token: "{{ config['api_key'] }}"
+          inject_into:
+            type: RequestOption
+            field_name: Api-Key
+            inject_into: header
+        error_handler:
+          type: CompositeErrorHandler
+          error_handlers:
+            - type: DefaultErrorHandler
+              response_filters:
+                - type: HttpResponseFilter
+                  action: IGNORE
+                  http_codes:
+                    - 401
```

Comment:
Given that we used to allow this stream to continue on 500s (based on the presence of the `test_listuser_stream_keep_working_on_500` test, should we have another error handler here to account for that?

Reply:
DEFAULT_RETRIABLE_ERRORS include 500s, so we don't need defined it in manifest